database/migrate_legacy_data_20220101.sh

```diff
@@ -0,0 +1,226 @@
+#!/bin/bash
+
+# This file handles the migration of user data, from the old folders and files
+# storage, to the new database model
+
+KW_LIB_DIR='src'
+KW_DB_DIR='database'
+. 'src/kw_include.sh' --source-only
+include "$KW_LIB_DIR/kwio.sh"
+include "$KW_LIB_DIR/kwlib.sh"
+include "$KW_LIB_DIR/kw_string.sh"
+include "$KW_LIB_DIR/kw_db.sh"
```

Comment:
Please, review the code style in this entire file to ensure it follows kw standard.